packages/swap/prisma/migrations/20250116152326_backfill_channel_data_to_swap_request/migration.sql

```diff
@@ -0,0 +1,18 @@
+-- backfill simple columns from deposit channel to swap request
+UPDATE "public"."SwapRequest" sr
+    SET "maxBoostFeeBps" = dc."maxBoostFeeBps", "totalBrokerCommissionBps" = dc."totalBrokerCommissionBps"
```

Comment:
TODO: add additional properties that were added to the `SwapRequest` model (eg `fokMinPriceX128`) before merging this